docs/pages/installation.rst

```diff
@@ -22,6 +22,22 @@ Install the package from the ``conda-forge`` channel:
 .. code-block:: bash
 
     conda install dask-sql -c conda-forge
+ 
+With ``GPU Support``
+--------------
```

Comment:
I think it makes sense to make this a subsection (by switching to `^^^`) of the Conda install section since this can really only be done with Conda at the moment:

```suggestion
Experimental GPU support
^^^^^^^^^^^^^^^^^^^^^^^^
```